lib/beaker-hostgenerator/util.rb

```diff
@@ -1,36 +1,24 @@
 require 'beaker-hostgenerator/data'
```

Comment:
@wayne Ok I restored this file, but had to refactor it as well.  I still wanted to move most of this stuff to Data, so the implementations are now mostly just forwarding.